### ISSUE TITLE
TypeErrors (expected list, got tuple) with python 3.12

### ISSUE THREAD

**Issue by ncopa** [jaraco/inflect#207]
When running the tests from latest available release (7.0.0) with python 3.12 I get those errors:

```
======================================= test session starts ========================================
platform linux -- Python 3.12.2, pytest-8.0.2, pluggy-1.4.0
rootdir: /home/ncopa/aports/community/py3-inflect/src/inflect-7.0.0
configfile: pytest.ini
plugins: mock-3.10.0, hypothesis-6.99.4
collected 0 items / 29 errors                                                                      

============================================== ERRORS ==============================================
_______________________________ ERROR collecting inflect/__init__.py _______________________________
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
________________________________ ERROR collecting tests/test_an.py _________________________________
tests/test_an.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
________________________________ ERROR collecting tests/test_an.py _________________________________
tests/test_an.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
___________________________ ERROR collecting tests/test_classical_all.py ___________________________
tests/test_classical_all.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
___________________________ ERROR collecting tests/test_classical_all.py ___________________________
tests/test_classical_all.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_________________________ ERROR collecting tests/test_classical_ancient.py _________________________
tests/test_classical_ancient.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_________________________ ERROR collecting tests/test_classical_ancient.py _________________________
tests/test_classical_ancient.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
__________________________ ERROR collecting tests/test_classical_herd.py ___________________________
tests/test_classical_herd.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
__________________________ ERROR collecting tests/test_classical_herd.py ___________________________
tests/test_classical_herd.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
__________________________ ERROR collecting tests/test_classical_names.py __________________________
tests/test_classical_names.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
__________________________ ERROR collecting tests/test_classical_names.py __________________________
tests/test_classical_names.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_________________________ ERROR collecting tests/test_classical_person.py __________________________
tests/test_classical_person.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_________________________ ERROR collecting tests/test_classical_person.py __________________________
tests/test_classical_person.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
__________________________ ERROR collecting tests/test_classical_zero.py ___________________________
tests/test_classical_zero.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
__________________________ ERROR collecting tests/test_classical_zero.py ___________________________
tests/test_classical_zero.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_____________________________ ERROR collecting tests/test_compounds.py _____________________________
tests/test_compounds.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_____________________________ ERROR collecting tests/test_compounds.py _____________________________
tests/test_compounds.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
____________________________ ERROR collecting tests/test_inflections.py ____________________________
tests/test_inflections.py:5: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
____________________________ ERROR collecting tests/test_inflections.py ____________________________
tests/test_inflections.py:5: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_______________________________ ERROR collecting tests/test_join.py ________________________________
tests/test_join.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_______________________________ ERROR collecting tests/test_join.py ________________________________
tests/test_join.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_____________________________ ERROR collecting tests/test_numwords.py ______________________________
tests/test_numwords.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_____________________________ ERROR collecting tests/test_numwords.py ______________________________
tests/test_numwords.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_______________________________ ERROR collecting tests/test_pl_si.py _______________________________
tests/test_pl_si.py:3: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
_______________________________ ERROR collecting tests/test_pl_si.py _______________________________
tests/test_pl_si.py:3: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
________________________________ ERROR collecting tests/test_pwd.py ________________________________
tests/test_pwd.py:3: in <module>
    from inflect import (
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
________________________________ ERROR collecting tests/test_pwd.py ________________________________
tests/test_pwd.py:3: in <module>
    from inflect import (
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
______________________________ ERROR collecting tests/test_unicode.py ______________________________
tests/test_unicode.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
______________________________ ERROR collecting tests/test_unicode.py ______________________________
tests/test_unicode.py:1: in <module>
    import inflect
inflect/__init__.py:2027: in <module>
    class engine:
inflect/__init__.py:2048: in engine
    @validate_call
pydantic/decorator.py:49: in pydantic.decorator.validate_arguments
    ???
pydantic/decorator.py:36: in pydantic.decorator.validate_arguments.validate
    ???
pydantic/decorator.py:126: in pydantic.decorator.ValidatedFunction.__init__
    ???
pydantic/decorator.py:264: in pydantic.decorator.ValidatedFunction.create_model
    ???
pydantic/main.py:1024: in pydantic.main.create_model
    ???
pydantic/main.py:186: in pydantic.main.ModelMetaclass.__new__
    ???
E   TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
===================================== short test summary info ======================================
ERROR inflect/__init__.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_an.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_an.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_all.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_all.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_ancient.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_ancient.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_herd.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_herd.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_names.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_names.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_person.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_person.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_zero.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_classical_zero.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_compounds.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_compounds.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_inflections.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_inflections.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_join.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_join.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_numwords.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_numwords.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_pl_si.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_pl_si.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_pwd.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_pwd.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_unicode.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
ERROR tests/test_unicode.py - TypeError: Argument 'bases' has incorrect type (expected list, got tuple)
!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 29 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================== 29 errors in 1.37s ========================================
```

I was not able to find anything evident int the commit log or issues.

**Comment by ncopa** [comment]
Seems to be an issue in pydantic: https://github.com/pydantic/pydantic/issues/7689

**Comment by ncopa** [comment]
I'm closing this because those appear to be fixed with pydantic 2.6.4.
